DataFormats/Math/interface/GraphUtil.h

```diff
@@ -0,0 +1,76 @@
+#ifndef DATA_FORMATS_MATH_GRAPH_UTIL_H
+#define DATA_FORMATS_MATH_GRAPH_UTIL_H
+
+#include "DataFormats/Math/interface/Graph.h"
+#include "DataFormats/Math/interface/GraphWalker.h"
+#include <iostream>
+#include <string>
+
+template<class N, class E>
+  void output(const math::Graph<N,E> & g, const N & root)
+{
+  math::GraphWalker<N,E> w(g,root);
+  bool go=true;
+  while(go) {
+    std::cout << w.current().first << ' ';
+    go=w.next();
+  }
+  std::cout << std::endl;
```

Comment:
there should probably be a MessageLogger-aware version, otherwise this interface is unusable in multithreaded mode.